package.json

```diff
@@ -29,7 +29,7 @@
       "VirtualMachinesList": "./views/virtualmachines/list/VirtualMachinesList.tsx",
       "TemplatesCatalog": "./views/catalog/templatescatalog/TemplatesCatalog.tsx",
       "VirtualMachinePage": "./views/virtualmachines/details/VirtualMachinePage.tsx",
-      "VirtualMachinesInstancesPage": "./views/virtualmachinesinstances/page/VirtualMachinesInstancesPage.tsx"
+      "VirtualMachinesInstancesPage": "./views/virtualmachinesInstance/details/VirtualMachinesInstancesPage.tsx"
```

Comment:
Well, I can see Matan really renamed the dir from _virtualmachinesinstances_ to _virtualmachinesInstance_, so I think it is not a typo. But yeah, it is a bit uncommon here as other dirs under _views_ are all lowercase. But IMHO it is more readable with big "I" here as the name _virtualmachinesInstance_ is longer.

Reply:
currently all directory names are lower case, it we want to change to camel case we need to discuss it, and it can't be done in this PR.

@pcbailey already requested that we move to camel case, and if we decide to do it, we will move all directories to camel case, not just this one.

@metalice for this PR, please don't change the directories structure

Reply:
i didnt change this typo this is the folder that was in the project virtualmachinesInstance , I removed the folder I added which was virtualmachinesinstances

Reply:
Ah okay. Yes currently we have two folders: `src/views/virtualmachinesInstance` `src/views/virtualmachinesinstances`.

Reply:
great, so can you fix the typo and change the dir to `virtualmachinesinstance` instead of `virtualmachinesInstance` ?
we should have a uniform directory structure, over all our code base, currently it's all lower case, meaning we need to fix this typo, and since this PR is about fixing the vmis directories, it's a good place to do it.

Reply:
> currently in main we have two folders: src/views/virtualmachinesInstance src/views/virtualmachinesinstances.

@upalatucci this PR is reverting https://github.com/kubevirt-ui/kubevirt-plugin/pull/75 that introduced a new directory by mistake, it's here to fix this two directories situation, and if we are here lets fix the typo in the dir name as well.